rabbus.go

```diff
@@ -66,9 +65,11 @@ type Message struct {
 	// Key the routing key name.
 	Key string
 	// Payload the message payload.
-	Payload interface{}
+	Payload []byte
```

Comment:
@FelipeUmpierre it should be a `[]byte` because amqp.Publishing object expects it on `Body`. Once the Rabbus class don't Marshall the information anymore, we need to make sure the client is sending this kind of data.